src/components/redaction/redaction-media-library-tab.js

```diff
@@ -18,15 +18,15 @@ import { useSessionAwareApiClient } from '../../ui/api-helper.js';
 import { mediaLibraryItemShape } from '../../ui/default-prop-types.js';
 import { confirmMediaFileHardDelete } from '../confirmation-dialogs.js';
 import { getResourceTypeTranslation } from '../../utils/resource-utils.js';
-import { ensureIsExcluded, replaceItem } from '../../utils/array-utils.js';
 import permissions, { hasUserPermission } from '../../domain/permissions.js';
 import MediaLibraryApiClient from '../../api-clients/media-library-api-client.js';
 import ActionButton, { ActionButtonGroup, ACTION_BUTTON_INTENT } from '../action-button.js';
-import MediaLibaryMetadataModal, { MEDIA_LIBRARY_METADATA_MODAL_MODE } from '../resource-selector/media-library/media-library-item-modal.js';
+import { ensureIsExcluded, ensureIsIncluded, replaceItem } from '../../utils/array-utils.js';
+import MediaLibaryItemModal, { MEDIA_LIBRARY_METADATA_MODAL_MODE } from '../resource-selector/media-library/media-library-item-modal.js';
 
 const logger = new Logger(import.meta.url);
 
-function getDocumentMetadataModalState({ mode = MEDIA_LIBRARY_METADATA_MODAL_MODE.preview, mediaLibraryItem = null, isOpen = false }) {
+function getMediaLibraryMetadataModalState({ mode = MEDIA_LIBRARY_METADATA_MODAL_MODE.create, mediaLibraryItem = null, isOpen = false }) {
```

Comment:
Uh, copy&paste all over 😬

Reply:
Corrected everywhere to "MediaLibraryItemModal..."